src/main/java/twofive/command/Command.java

```diff
@@ -0,0 +1,28 @@
+package twofive.command;
+
+import twofive.data.TaskList;
+import twofive.exception.TwoFiveException;
+import twofive.storage.Storage;
+
+/**
+ * Results in various actions being performed when commands are executed
+ * in initialized subclasses of this class.
+ */
```

Comment:
Great job going above and beyond the minimal requirements in adding separate classes for Commands